### PR TITLE
fix: updating dependabot yml to include prefix for PR rasied by it

### PR DESCRIPTION
**Pull request by amopatel1234**
# Add Prefix to dependabot PRs

Updated dependabot yml file to include a prefix to commit message when it raise PR for updating swift packages and actions

# Checklist

## Before raising your pull request:
- [ ] ~Ran the app locally ensuring it builds ~
- [ ] ~Ran the tests locally ensuring they pass on Build~
- [x] Pull request has a clear title with ticket ID and a short description about the feature or update
      i.e. _DCMAW-222: Added ReadID SDK to iOS app_
- [ ] ~Created a `draft` pull request if it is not yet ready for review~

## Before your pull request can be reviewed:
- [ ] ~Met all of the acceptance criteria specified in the user story on Jira~
- [ ] ~Reviewed your own code to ensure you are following the style guidelines~
- [ ] ~Ran the app and tested the feature on a range of device sizes~
      ~Please include iPod Touch, iPhone SE and iPhone 11 as a minimum.~
- [ ] ~Written Unit and Integration tests if needed~

- [ ] ~Met all accessibility requirements?~
    - [ ] ~Checked dynamic type sizes are applied~
    - [ ] ~Checked VoiceOver can navigate your new code~
    - [ ] ~Checked a user can navigate only using a keyboard around your new code~

## Before merging your pull request:
- [ ] Ensure that the code coverage and SonarCloud checks have passed
- [ ] Actioned and resolved all comments, reaching out to reviewers for clarifications if necessary.
- [ ] Ran the app to ensure that no regressions have been caused by changes during code review.
- [ ] Targeted the correct branch; `develop`, `release` or `main`

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,8 +7,12 @@ updates:
...
+      prefix: "chore"
   - package-ecosystem: "swift"
     directory: "/"
     schedule:
       interval: "weekly"
-    open-pull-requests-limit: 5
\ No newline at end of file
+    commit-message:
+      prefix: "chore"
+    open-pull-requests-limit: 5
```
Comment (DylanRJ) [inline .github/dependabot.yml:18]: Can we make this "build"? https://github.com/Oliver-Binns/Versioning/pull/13

Comment (amopatel1234) [reply .github/dependabot.yml:18]: have changed this now to build

File: .github/dependabot.yml [hunk 2]
```diff
@@ -8,11 +8,11 @@ updates:
       # Check for updates to GitHub Actions every week
       interval: "weekly"
     commit-message:
-      prefix: "chore"
+      prefix: "build"
```
Comment (DylanRJ) [inline .github/dependabot.yml:11]: In my mind, for github actions updates, it's fine to have them as "chore" or "ci"? We only care about incrementing the patch version when the code dependencies are updated?

**Comment by DylanRJ** [comment]
Going to close this in preference of #70 if you don't mind

**Pull request closed without merging**